options/locale/locale_cs-CZ.ini

```diff
@@ -2235,33 +2235,6 @@ find_file.no_matching=Nebyl nalezen žádný odpovídající soubor
 error.csv.too_large=Tento soubor nelze vykreslit, protože je příliš velký.
 error.csv.unexpected=Tento soubor nelze vykreslit, protože obsahuje neočekávaný znak na řádku %d ve sloupci %d.
 error.csv.invalid_field_count=Soubor nelze vykreslit, protože má nesprávný počet polí na řádku %d.
-filter_branch_and_tag=Filtr pro větev nebo značku
-find_tag=Najít značku
-branches=Větve
-tags=Značky
-issues=Úkoly
-pulls=Požadavky na natažení
-project_board=Projekty
-packages=Balíčky
-labels=Štítky
-org_labels_desc=Štítky na úrovni organizace, které mohou být použity se <strong>všemi repozitáři</strong> v rámci této organizace
-org_labels_desc_manage=spravovat
-milestones=Milníky
-commits=Commity
-commit=Commit
-release=Vydání
-releases=Vydání
-tag=Značka
-released_this=vydal/a toto
-file.title=%s v %s
-file_raw=Surový
-file_history=Historie
-file_view_source=Zobrazit zdroj
-file_view_rendered=Zobrazit vykreslené
-file_view_raw=Zobrazit v surovém stavu
-file_permalink=Trvalý odkaz
-file_too_large=Soubor je příliš velký pro zobrazení.
-invisible_runes_header=Tento soubor obsahuje neviditelné znaky Unicode!
```

Comment:
These removals are correct, there are duplicate keys above. This incorrect section was caused by broken quoting.